ndk/CHANGELOG.md

```diff
@@ -19,6 +19,7 @@
 - Fixed `HardwareBuffer` leak on buffers returned from `AndroidBitmap::get_hardware_buffer()`. (#296)
 - **Breaking:** Update `jni` crate (used in public API) from `0.18` to `0.19`. (#300)
 - hardware_buffer: Made `HardwareBufferDesc` fields `pub`. (#313)
+- **Breaking:** Update `raw-window-handle` to `v0.5` removing `HasRawWindowHandle` methods.
```

Comment:
Not a breaking change because it was only added in the same release, see:

```
- Implement `HasRawWindowHandle` directly on `NativeWindow`. (#274)
```

right above.

Can you add `, #319` to it though?

Reply:
Sure.